gremlin-go/README.md

```diff
@@ -91,65 +121,190 @@ Note: The exact import name as well as the module prefix for `NewDriverRemoteCon
 remote graph:
-<!--
-TODO: Add Go code example of a Gremlin traversal query.
--->
+```go
+package main
+
+import (
+	"fmt"
+	"github.com/lyndonb-bq/tinkerpop/gremlin-go/driver"
+)
 
-# The following material is currently Work-in-progress: 
+func main() {
+	// Creating the connection to the server.
+	driverRemoteConnection, err := gremlingo.NewDriverRemoteConnection("localhost", 8182)
+	// Handle error
+	if err != nil {
+		fmt.Println(err)
+		return
+	}
+	// Create an anonymous traversal source with remote
+	g := gremlingo.Traversal_().WithRemote(driverRemoteConnection)
+
+	// Add a vertex with properties to the graph with the terminal step Iterate()
+	_, promise, err := g.AddV("gremlin").Property("language", "go").Iterate()
+	if err != nil {
+		fmt.Println(err)
+		return 
+	}
+	// The returned promised is a go channel, to wait for all submitted steps to finish execution.
+	<-promise
+	
+	// Get the value of the property
+	result, err := g.V().HasLabel("gremlin").Values("language").ToList()
+	if err != nil {
+		fmt.Println(err)
+		return 
+	}
+	// Print the result
+	for _, r := range result {
+		fmt.Println(r.GetString())
+	}
+}
+```
 
 ## Sample Traversals
 <!--
 TODO: Add Go specific changes to following paragraph:
-examples:
-"For the most part, these examples should generally translate to Go with [little modification][differences]"
-"Given the strong correspondence between canonical Gremlin in Java and its variants like Go, there is a limited amount 
-of Go-specific documentation and examples."
 -->
 The Gremlin language allows users to write highly expressive graph traversals and has a broad list of functions that
-cover a wide body of features. The [Reference Documentation][steps] describes these functions and other aspects of the
+cover a wide body of features. Traversal in `Go` is very similar to other GLV, with the exception that all step functions are capitalized.
+
+<!--
+The [Reference Documentation][steps] describes these functions and other aspects of the
 TinkerPop ecosystem including some specifics on [Gremlin in Go][docs] itself. Most of the examples found in the
 documentation use Groovy language syntax in the [Gremlin Console][console]. For the most part, these examples
 should generally translate to Go with [little modification][differences]. Given the strong correspondence
 between canonical Gremlin in Java and its variants like Go, there is a limited amount of Go-specific
 documentation and examples. This strong correspondence among variants ensures that the general Gremlin reference
 documentation is applicable to all variants and that users moving between development languages can easily adopt the
 Gremlin variant for that language.
-
-### Create Vertex
-<!--
-TODO: Add Go code to create a vertex.
 -->
 
+### Create Vertex
+Adding a vertex with properties.
+```go
+_, promise, err :=g.AddV("gremlin").Property("language", "java").Iterate()
```

Comment:
1. nitpick: missing space between `:=` and `g.AddV`
2. Should the language here still be `java`?